drivers/usb/host/xhci-pci.c

```diff
@@ -307,7 +307,6 @@ static void xhci_pci_quirks(struct device *dev, struct xhci_hcd *xhci)
 
 	if (pdev->vendor == PCI_VENDOR_ID_VIA && pdev->device == 0x3483) {
 		xhci->quirks |= XHCI_LPM_SUPPORT;
-		xhci->quirks |= XHCI_EP_CTX_BROKEN_DCS;
```

Comment:
I'm confused. If this is a revert revert of `xhci: add quirk for host controllers that don't update endpoint DCS` then shouldn't it look similar to the original  `xhci: add quirk for host controllers that don't update endpoint DCS` commit?

Yet,  `xhci: add quirk for host controllers that don't update endpoint DCS` adds the line `xhci->quirks |= XHCI_EP_CTX_BROKEN_DCS` and this is removing it? (which sounds like a single revert?)

Reply:
No, I'm the confused one. It appears that that line wasn't removed by the initial revert, even though it was added by the original commit - either a problem fixing up the revert or something strange with merge commits. It should be OK now.